### DISCUSSION TITLE
Linear XP incompatibility with Global XP mod

### DISCUSSION THREAD

**Discussion by Foreck1** [ACGaming/UniversalTweaks#396] [Bugfix Suggestions]
When storing XP in the [Global XP ](https://www.curseforge.com/minecraft/mc-mods/global-xp) block, levels will calculate differently within, leading to a serious XP dupe if linear XP is enabled

**Comment by NoMoreUsernames999** [comment]
This feature was requested for my pack and so was made to specifically have compatibility with "bottle your xp." It seems to be incompatible with pretty much anything else that handles xp and breaks with other xp storage mods. Also breaks vanilla enchanting fyi. Is RoTN going to use this feature for something??
